x-pack/plugins/siem/public/common/components/exceptions/viewer/index.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPanel, EuiFlexGroup, EuiCommentList, EuiAccordion, EuiFlexItem } from '@elastic/eui';
+import React, { useEffect, useState, useMemo, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { ExceptionDetails } from './exception_details';
+import { ExceptionEntries } from './exception_entries';
+import { getFormattedEntries, getFormattedComments } from '../helpers';
+import { FormattedEntry, ExceptionListItemSchema } from '../types';
+
+const MyFlexItem = styled(EuiFlexItem)`
+  &.comments--show {
+    padding: 16px;
+    border-top: 1px solid #d3dae6;
+  }
+`;
+
+interface ExceptionItemProps {
+  exceptionItem: ExceptionListItemSchema;
+  handleDelete: ({ id }: { id: string }) => void;
+  handleEdit: (item: ExceptionListItemSchema) => void;
+}
+
+const ExceptionItemComponent = ({
+  exceptionItem,
+  handleDelete,
+  handleEdit,
+}: ExceptionItemProps) => {
+  const [entryItems, setEntryItems] = useState<FormattedEntry[]>([]);
+  const [showComments, setShowComments] = useState<boolean>(false);
```

Comment:
Nit: technically you don't have to use a template here and it will derive it from the false you pass in:

```ts
= useState(false);
```

For empty arrays or nulls, etc... you do have to add it as a template because otherwise it cannot figure out which empty array or null the type of state belongs to. Totally optional here. Just pointing it out.

Reply:
I got type happy 😀  Updated to remove unnecessary typing. Thanks!